### PR TITLE
LPD-25260: Add the missing OpenSearch test properties

### PR DESCRIPTION
**Pull request by peerkar**
https://liferay.atlassian.net/browse/LPD-25260

### CODE REVIEW

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by gustavolimav** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:reevaluate:1277401_11860

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:stop

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:stop

**Comment by peerkar** [comment]
ci:test:search-opensearch2

File: build-test-batch.xml [hunk 1]
```diff
@@ -964,6 +964,8 @@ mariadb.executable=${mariadb.executable}]]></echo>
 				<then>
 					<ant antfile="build-test-opensearch2.xml" target="start-opensearch" />
 
+					<property name="test.batch.name" unless:set="test.batch.name" value="${env.TEST_BATCH_NAME}" />
```
Comment (kenjiheigel) [inline build-test-batch.xml:967]: @peerkar, Move this addition to line `945` above.

**Comment by peerkar** [comment]
ci:test:search-opensearch2

**Comment by peerkar** [comment]
ci:test:sf

**Comment by peerkar** [comment]
ci:stop

**Comment by peerkar** [comment]
Tests not triggered. Continuing [here](https://github.com/liferay-search/liferay-portal/pull/1354)

**Pull request closed without merging**